scripts/cf-operator-delete.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+source scripts/include/setup.sh
+
+require_tools kubectl helm
+
+set +e
```

Comment:
Nit: I would prefer to `set -e`, and use

```
if helm_ls 2>/dev/null | grep -qi cf-operator ; then
    helm_delete cf-operator --namespace cf-operator
fi
kubectl delete --ignore-not-found ns "${CF_OPERATOR_NS}"
```

and so on

Reply:
With the new changes, we can drop the `set +e`. I would like to do `set -e` in scripts/include/setup.sh to catch errors in the scripts in the future, but that's out of this PR.